### PR TITLE
Feat manual beer meter

### PR DESCRIPTION
**Pull request by DomantasDev**
I implemented manual beer meter feature, check the code and run the App to see if you agree with this.

### CODE REVIEW

File: BeerOverflowWindowsApp/BeerPanel.cs [hunk 1]
```diff
@@ -0,0 +1,95 @@
...
+                mousePressed = true;
+            }
+        }
+
+        private void BeerPanel_MouseMove(object sender, MouseEventArgs e)
+        {
+            if (mousePressed)
+            {
+                rectangle.Y = e.Y - difference;
+                if (rectangle.Y + rectangle.Height / 2 < top.Y)
```
Comment (joklek) [inline BeerOverflowWindowsApp/BeerPanel.cs]: Can you explain what this bit of code is doing?

File: BeerOverflowWindowsApp/BeerPanel.cs [hunk 2]
```diff
@@ -0,0 +1,95 @@
...
+        private void BeerPanel_MouseDown(object sender, MouseEventArgs e)
+        {
+            if (rectangle.Contains(e.X, e.Y))
+            {
+                difference = e.Y - rectangle.Y;
+                mousePressed = true;
+            }
+        }
+
+        private void BeerPanel_MouseMove(object sender, MouseEventArgs e)
```
Comment (joklek) [inline BeerOverflowWindowsApp/BeerPanel.cs]: Can you explain the math you're doing in this function?

Comment (DomantasDev) [reply BeerOverflowWindowsApp/BeerPanel.cs]: First of all, all calculations are done only with Y coordinates. The "rectangle" variable actually represents the position of the marker(going to rename that). "rectangle.Y + rectangle.Height / 2" is used to determine the Y coordinate of the middle of the marker and those two if's are just for not letting the marker to go above "top" point or below "bottom" point. 
"rating = (bottom.Y - rectangle.Y - rectangle.Height / 2) / (height / 100)" is calculating the percentage value of how much of the glass is filled (beer height / glass height * 100 just in a little bit different form)

**Comment by joklek** [comment]
[code here](https://github.com/joklek/BeerOverflowWindowsApp/blob/e470b0c3f551b8b6c8a0e779d13c3e6a6f1f84fb/BeerOverflowWindowsApp/BeerPanel.cs#L19) 

    private static string rating;

    public static string Rating

    {

        get { return rating; }

    }

could be changed into for cleaner code, I think

    public static string Rating { get; set; }

**Comment by DomantasDev** [comment]
I see what you mean, I will change it into
 `public static string Rating {get; private set;}`

**Pull request merged**